packages/server/src/index.ts

```diff
@@ -0,0 +1,156 @@
+/**
+ * @packageDocumentation
+ *
+ * Implements HTTP routes for a Fastify server that conform to the [IPFS Pinning Service API](https://ipfs.github.io/pinning-services-api-spec/).
+ *
+ * @example
+ *
+ * ```typescript
+ * import { createHelia } from 'helia'
+ * import { createPinningServiceAPIServer} from '@helia/pinning-service-api-server'
+ *
+ * const helia = await createHelia()
+ * const server = await createRemotePinningServer(helia, {
+ *   validateAccessToken: async (accessToken, options) => {
+ *     // validate the user passed a valid token
+ *     return true
+ *   },
+ *   listen: {
+ *     // fastify listen options
+ *   }
+ * })
+ *
+ * // now make http requests
+ * ```
+ *
+ * Alternatively if you have a Fastify instance already you can add routes to it.
+ *
+ * @example
+ *
+ * ```typescript
+ * import fastify from 'fastify'
+ * import cors from '@fastify/cors'
+ * import { createHelia } from 'helia'
+ * import routes from '@helia/pinning-service-api-server/routes'
+ *
+ * const server = fastify({
+ *  // fastify options
+ * })
+ * await server.register(cors, {
+ *   origin: '*',
+ *   methods: ['GET', 'OPTIONS'],
+ *   strictPreflight: false
+ * })
+ *
+ * const helia = await createHelia()
+ *
+ * // configure Routing V1 HTTP API routes
```

Comment:
routing v1 names should be changed to "pinning api" or similar

Reply:
Done!